src/logic/operate.js

```diff
@@ -0,0 +1,25 @@
+import Big from 'big.js';
+
+const Operate = (() => {
+  const operate = (numberOne, numberTwo, operation) => {
+    // eslint-disable-next-line
+    let total;
+    const one = Big(numberOne);
+    const two = Big(numberTwo);
+    if (operation === '-') {
+      total = one - two;
+    } else if (operation === '+') {
+      total = one + two;
+    } else if (operation === 'X') {
+      total = one * two;
+    } else if (operation === '÷') {
+      total = one / two;
+    } else if (operation === '%') {
+      total = (one / 100) * two;
+    }
```

Comment:
- [ ] Make sure to test if the modifier operation works when only number one was typed.